test/integration/util/util.go

```diff
@@ -217,6 +217,22 @@ func CleanupTest(t *testing.T, testCtx *TestContext) {
 	testCtx.CloseFn()
 }
 
+// RemovePodFinalizers removes pod finalizers for the pods
+func RemovePodFinalizers(cs clientset.Interface, t *testing.T, pods []*v1.Pod) {
+	for _, p := range pods {
+		pod, err := cs.CoreV1().Pods(p.Namespace).Get(context.TODO(), p.Name, metav1.GetOptions{})
+		if err != nil && !apierrors.IsNotFound(err) {
+			t.Errorf("error while removing pod finalizers for %v: %v", klog.KObj(p), err)
+		} else if pod != nil && err == nil {
```

Comment:
This should be enough, right?

```suggestion
		} else if pod != nil {
```